data/hai/hai reveal 4 middle.txt

```diff
@@ -278,7 +278,49 @@ mission "Pirate Troubles [0]"
 
 
 
-mission "Pirate Troubles [1]"
+# If older mission chain was done: Need to remove the planet the asteroid orbits, then re-add it without the landable location to reset.
+mission "Hai Reveal: Pirate Troubles [0a]"
+	invisible
+	source
+		near "Waypoint" 3
+		government "Hai"
+		not planet "Allhome"
```

Comment:
No need to have the same source as PT [0], just trigger this anywhere when we know we'll need it.
```suggestion
	landing
	invisible
```